project_app/components/Header.js

```diff
@@ -7,100 +7,98 @@ import Image from "next/image";
-              position: "absolute",
-              width: "244px",
-              height: "109px",
-              left: "477px",
-              top: "560px",
-              bottom: "78.57%",
-              zIndex: "0",
-            }}
-          >
-            <Image src="/people.png" fill="absolute" alt="image" />
-          </div>
-          <div
-            style={{
-              position: "absolute",
-              width: "90.37px",
-              height: "105.71px",
-              left: "708.46px",
-              top: "557.57px",
-            }}
-          >
-            <Image src="/fire.png" fill="absolute" alt="image" />
-          </div>
-          <div
-            style={{
-              position: "absolute",
-              width: "651.27px",
-              height: "464.65px",
-              left: "789.04px",
-              bottom: "0",
-              top: "217.54px",
-              right: "0",
-              zIndex: "0",
-            }}
-          >
-            <Image src="/trees_right.png" fill="none" alt="image" />
-          </div>
-        </>
-      </AppBar>
-    </>
+        </div>
+        <div>
+          <ToolBar />
+        </div>
+        <div>
+          <NavBarButtons />
+        </div>
+      </Container>
+      <>
+        <Typography>Interactive BoardGame Community</Typography>
+        <div
+          style={{
+            position: "absolute",
+            width: "629.05px",
+            height: "471.51px",
+            top: "204px",
+            left: "0",
+            bottom: "0",
+          }}
+        >
+          <Image src="/trees_left.png" fill="none" alt="image" />
+        </div>
+        <div
+          style={{
+            position: "absolute",
+            width: "296px",
+            height: "250px",
+            left: "412.31px",
+            top: "422px",
+          }}
+        >
+          <Image src="/Tent.png" fill="absolute" alt="image" />
+        </div>
+        <div
+          style={{
+            position: "absolute",
+            width: "244px",
+            height: "109px",
+            left: "477px",
+            top: "560px",
+            bottom: "78.57%",
+            zIndex: "0",
+          }}
+        >
+          <Image src="/people.png" fill="absolute" alt="image" />
+        </div>
+        <div
+          style={{
+            position: "absolute",
+            width: "90.37px",
+            height: "105.71px",
+            left: "708.46px",
+            top: "557.57px",
+          }}
+        >
+          <Image src="/fire.png" fill="absolute" alt="image" />
+        </div>
+        <div
+          style={{
+            position: "absolute",
+            width: "651.27px",
+            height: "464.65px",
+            left: "789.04px",
+            bottom: "0",
+            top: "217.54px",
+            right: "0",
+            zIndex: "0",
+          }}
+        >
+          <Image src="/trees_right.png" fill="none" alt="image" />
+        </div>
+      </>
+    </AppBar>
```

Comment:
lets move this to styled components. next sprint

Reply:
Ok sounds good